templates/visuals/rvisual/src/settings.ts

```diff
@@ -26,7 +26,32 @@
 
 "use strict";
 
-import { dataViewObjectsParser } from "powerbi-visuals-utils-dataviewutils";
-import DataViewObjectsParser = dataViewObjectsParser.DataViewObjectsParser;
+import { formattingSettings } from "powerbi-visuals-utils-formattingmodel";
 
-<%= visualSettingsClasses %>
+import FormattingSettingsCard = formattingSettings.Card;
+import FormattingSettingsSlice = formattingSettings.Slice;
+import FormattingSettingsModel = formattingSettings.Model;
+
+
+/**
+ * RCV Script Formatting Card
+ */
+class rcvScriptCardSettings extends FormattingSettingsCard {
+    provider: FormattingSettingsSlice = undefined;
```

Comment:
I added here undefined cards because it was the same (slices values are undefined) when i ran `pbiviz new -t rvisual`